src/main/scala/org/broadinstitute/gpp/poolq3/PoolQConfig.scala

```diff
@@ -96,6 +96,7 @@ final case class PoolQConfig(
   unexpectedSequenceCacheDir: Option[Path] = None,
   removeUnexpectedSequenceCache: Boolean = true,
   unexpectedSequencesToReport: Int = 100,
+  unexpectedSequenceSamplePct: Float = 0.02f,
```

Comment:
opinion: I don't think there's any reason to go with `Float` instead of `Double` here.   Since `Double` is the default I think it should be always used unless you really need to optimize for space in some data object that's going to be instantiated by the millions.